src/main/g8/src/main/scala/$package$/QuickstartServer.scala

```diff
@@ -20,16 +21,17 @@ object QuickstartServer extends App with UserRoutes {
   //#server-bootstrapping
 
   val userRegistryActor: ActorRef = system.actorOf(UserRegistryActor.props, "userRegistryActor")
+  val bindingAddress = envOrElse("bindingAddress", "localhost")
 
   //#main-class
   // from the UserRoutes trait
   lazy val routes: Route = userRoutes
   //#main-class
 
   //#http-server
-  Http().bindAndHandle(routes, "localhost", 8080)
+  Http().bindAndHandle(routes, bindingAddress, 8080)
 
-  println(s"Server online at http://localhost:8080/")
+  println(s"Server online at http://\$bindingAddress:8080/")
```

Comment:
This will not show up correctly in the paradox-generated documentation unfortunately.

Reply:
Correct. I agree with you. I may require your inputs on how to solve this one.